image/discourse_test/Dockerfile

```diff
@@ -17,23 +17,24 @@ RUN chown -R discourse . &&\
 
 FROM base AS with_browsers
 
-RUN wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | sudo apt-key add - &&\
-    echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list &&\
-    apt update &&\
-    apt install -y libgconf-2-4 libxss1 google-chrome-stable firefox-esr &&\
+ADD install-chrome /tmp/install-chrome
+RUN /tmp/install-chrome
+
+RUN apt update &&\
+    apt install -y libgconf-2-4 libxss1 firefox-esr &&\
     cd /tmp && wget -q "https://download.mozilla.org/?product=firefox-latest-ssl&os=linux64&lang=en-US" -O firefox.tar.bz2 &&\
     tar xjvf firefox.tar.bz2 && mv /tmp/firefox /opt/firefox-evergreen &&\
     apt clean
 
 FROM with_browsers AS release
 
 RUN cd /var/www/discourse &&\
-    sudo -u discourse bundle install --jobs=4 &&\
```

Comment:
In the base image which this image builds upon, we have configured bundler `--jobs` to default to number of CPU cores minus 1.